src/include/distributed/master_metadata_utility.h

```diff
@@ -67,6 +67,7 @@ typedef struct ShardInterval
 	Datum minValue;     /* a shard's typed min value datum */
 	Datum maxValue;     /* a shard's typed max value datum */
 	uint64 shardId;
+	int shardIndex;
```

Comment:
Don't we need to add this to `CopyNodeShardInterval()` and similar related functions?

Reply:
Added